requirements/test.txt

```diff
@@ -113,3 +112,6 @@ virtualenv==16.6.0        # via tox
 wcwidth==0.1.7            # via prompt-toolkit
 whitenoise==3.3.0
 wrapt==1.11.1
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools==41.0.1        # via ipdb, ipython, pytest, tox
```

Comment:
Was this auto-added too? Should we remove it from the requirements file and require it somewhere else if it's really needed?

Reply:
Discussed as a team in Slack. There's a little bit of chatter here too: https://github.com/jazzband/pip-tools/issues/522. The follow up is to log a new issue to refine our use of pip-compile some more to catch things like this that may be undesirable.